rolling/distribution.yaml

```diff
@@ -1629,6 +1629,21 @@ repositories:
       url: https://github.com/ros2/message_filters.git
       version: master
     status: maintained
+  message_synchronizer:
+    doc:
+      type: git
+      url: https://github.com/OUXT-Polaris/message_synchronizer.git
```

Comment:
Looking at the [package.xml](https://github.com/OUXT-Polaris/message_synchronizer/blob/master/package.xml), it doesn't declare a dependency on boost, which means this is almost certainly going to fail to build on the buildfarm.  I'll suggest adding the appropriate boost dependency keys to the package.xml